org.jackhuang.hmcl.yml

```diff
@@ -0,0 +1,74 @@
+id: org.jackhuang.hmcl
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.openjdk17
+rename-desktop-file: HMCL.desktop
+rename-appdata-file: HMCL.appdata.xml
```

Comment:
since you ship these file, why not naming them properly?

Reply:
I think it's convenient if develop want to change id.  
And it works fine, so I think it's ok.

Reply:
there are in this repository. And on flathub the app-id is set in stone. Changing it require resubmitting the package.

This is only useful if you take them form upstream, which you should if they are there.